app/src/main/java/com/duckduckgo/app/settings/db/SettingsDataStore.kt

```diff
@@ -91,13 +95,9 @@ class SettingsSharedPreferences @Inject constructor(
         get() = preferences.getBoolean(KEY_AUTOCOMPLETE_ENABLED, true)
         set(enabled) = preferences.edit { putBoolean(KEY_AUTOCOMPLETE_ENABLED, enabled) }
 
-    override var appLoginDetection: Boolean
-        get() = if (variantManager.isFireproofExperimentEnabled()) {
-            preferences.getBoolean(KEY_LOGIN_DETECTION_ENABLED, false)
-        } else {
-            preferences.getBoolean(KEY_LOGIN_DETECTION_ENABLED, true)
-        }
-        set(enabled) = preferences.edit { putBoolean(KEY_LOGIN_DETECTION_ENABLED, enabled) }
+    override var appLoginDetection: LoginDetectorSetting
```

Comment:
`variantManager` should be removed from the constructor as well, it is no longer used